### PR TITLE
feat: init docker alpine tests

### PR DESCRIPTION
**Pull request by pantharshit00**
This PR introduces a new docker folder which shall container all of our docker related tests. 

Inside of that folder, we have a test for alpine linux for now. Mostly reusing artefacts from the pm2 test.

### CODE REVIEW

**Comment by pantharshit00** [comment]
Actually, ready for merge. Tests failure are expected.

File: docker/alpine/schema.prisma [hunk 1]
```diff
@@ -0,0 +1,14 @@
+generator client {
```
Comment (Jolg42) [inline docker/alpine/schema.prisma]: Why 2 schemas? one in docker/alpine/schema.prisma and one in docker/alpine/prisma/schema.prisma

**Comment by Jolg42** [comment]
Both library and engine jobs are failing with 
```
Error: Unable to require(`/usr/src/app/node_modules/@prisma/engines/libquery_engine-linux-musl.so.node`)
```

I expected the engine job the fail with the binary engine 🤔

File: docker/alpine/schema.prisma (hunk 1 above)
Comment (pantharshit00) [reply docker/alpine/schema.prisma]: Copy/paste from other e2e test. I deleted the extra one.

**Comment by pantharshit00** [comment]
> Both library and engine jobs are failing with
> 
> ```
> Error: Unable to require(`/usr/src/app/node_modules/@prisma/engines/libquery_engine-linux-musl.so.node`)
> ```
> 
> I expected the engine job the fail with the binary engine 🤔


Good catch! Looks like I forgot to pass the engine type env variable to docker. 

Edit: Looks like CLI still tried to open the library file regardless of the environment variable.

**Comment by Jolg42** [comment]
@pantharshit00 I think you'll need to use `PRISMA_CLI_QUERY_ENGINE_TYPE` (we use that in prisma/prisma)

**Review by janpio — APPROVED** [review]

**Pull request merged**